GitExtUtils/GitUI/UIExtensions.cs

```diff
@@ -43,5 +45,27 @@ public static bool IsFixedWidth(this Font ft, Graphics g)
 
             return fixedWidth;
         }
+
+        public static string? FormatBodyAndNotes(string? body, string? notes)
```

Comment:
Consider
```suggestion
        internal static string FormatBodyAndNotes(string bodyOrSubject, string? notes)
```
because it is to be called with `revision.Body ?? revision.Subject`.